x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/build_signals_query.ts

```diff
@@ -30,7 +30,7 @@ export const buildSignalsSearchQuery = ({ ruleId, index, from, to }: BuildSignal
           {
             range: {
               '@timestamp': {
-                gte: from,
+                gt: from,
```

Comment:
Just commenting to verify the switch from `gte` to `gt` as the timestamp supplied to this query:

https://github.com/elastic/kibana/blob/7e8273001406d2ecc8bb88b7e603bb56fd82937b/x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.ts#L46

is the `previousStartedAt` (or previous interval if n/a), which is the start of the last rule run, correct?

Reply:
correct :)